package.json

```diff
@@ -24,6 +24,7 @@
     "gulp-istanbul": "^0.3.0",
     "gulp-mocha": "^1.1.1",
     "gulp-coveralls": "^0.1.3",
-    "proxyquire": "^1.0.1"
+    "proxyquire": "^1.0.1",
+    "child-proc": "^0.0.1"
```

Comment:
This dep is for the last test for logManager. I wanted to remove the dir 'harfiles' recursively before executing to make sure it creates a dir when it doesn't already exist. That part can be removed if we always run the tests with a clean slate.